tests/environment.ts

```diff
@@ -8,23 +8,10 @@ export const MEMO_CAP: number = 21;
 export const ANON_APP_ID: string = "anon";
 export const MEMO_TEMPLATE = `1-${ANON_APP_ID}-`;
 
-export const INTEGRATION_ENVIRONMENT = integEnv();
```

Comment:
why it was removed? tests env should rely on env vars and be configured on CI to our internal testnet